modules/schematics/src/container/index.ts

```diff
@@ -83,21 +81,12 @@ function addStateToComponent(options: Partial<ContainerOptions>) {
 
     const componentClass = source.statements.find(
       (stm) => stm.kind === ts.SyntaxKind.ClassDeclaration
-    );
-    const component = componentClass as ts.ClassDeclaration;
-    const componentConstructor = component.members.find(
-      (member) => member.kind === ts.SyntaxKind.Constructor
-    );
-    const cmpCtr = componentConstructor as ts.ConstructorDeclaration;
-    const { pos } = cmpCtr;
-    const constructorText = cmpCtr.getText();
-    const [start, end] = constructorText.split('()');
-    const storeConstructor = [start, `(private store: Store)`, end].join('');
+    ) as ts.ClassDeclaration;
     const constructorUpdate = new ReplaceChange(
       componentPath,
-      pos,
-      `  ${constructorText}\n\n`,
-      `\n\n  ${storeConstructor}`
+      componentClass.members.pos,
+      '\n',
+      `\n  constructor(private store: Store) {}`
```

Comment:
In v15, the component schematic doesn't generate the constructor and `ngOnInit` method anymore. Instead, it generates an empty component class.